code/game/mecha/combat/marauder.dm

```diff
@@ -54,7 +54,7 @@
 	health = 550
 	wreckage = /obj/structure/mecha_wreckage/seraph
 	internal_damage_threshold = 20
-	force = 55
+	force = 72

```

Comment:
it was already a 2-hit-kill.

But dun do this, it's closed. Do the generalized damage multiplier config thing